x-pack/plugins/dataset_quality/common/translations.ts

```diff
@@ -83,3 +83,77 @@ export const flyoutIntegrationNameText = i18n.translate(
     defaultMessage: 'Name',
   }
 );
+
+/*
+Summary Panel
+*/
+
+export const summaryPanelLast24hText = i18n.translate(
+  'xpack.datasetQuality.summaryPanelLast24hText',
+  {
+    defaultMessage: 'Last 24h',
+  }
+);
+
+export const summaryPanelQualityText = i18n.translate(
+  'xpack.datasetQuality.summaryPanelQualityText',
+  {
+    defaultMessage: 'Datasets Quality',
+  }
+);
+
+export const summaryPanelQualityTooltipText = i18n.translate(
+  'xpack.datasetQuality.summaryPanelQualityTooltipText',
+  {
+    defaultMessage: 'Quality is based on the percentage of malformed docs in a dataset.',
```

Comment:
```suggestion
    defaultMessage: 'Quality is based on the percentage of degraded docs in a dataset.',
```

Reply:
I just saw 😅 but please change it because we renamed [from malformed to degraded docs](https://github.com/elastic/kibana/pull/172462#issuecomment-1840881365) the column name